.github/workflows/doc.yml

```diff
@@ -11,9 +11,9 @@ jobs:
         with:
           submodules: recursive
       - name: Set up Python 3.7
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v5
```

Comment:
Should this be part of a separate commit (maybe the previous one on documentation) and a relevant message be added to the commit?